dm/syncer/validator_checkpoint.go

```diff
@@ -519,13 +568,14 @@ func (c *validatorPersistHelper) loadError(filterState pb.ValidateErrorState) ([
 	args := []interface{}{
 		c.cfg.SourceID,
 	}
-	sql := "SELECT (id, source, src_schema_name, src_table_name, dst_schema_name, dst_table_name, data, dst_data, error_type, status, update_time) " +
+	query := "SELECT (id, source, src_schema_name, src_table_name, dst_schema_name, dst_table_name, data, dst_data, error_type, status, update_time) " +
 		"FROM " + c.errorChangeTableName + " WHERE source=?"
 	if filterState != pb.ValidateErrorState_InvalidErr {
-		sql += " AND status=?"
+		query += " AND status=?"
 		args = append(args, int(filterState))
 	}
-	rows, err = c.dbConn.QuerySQL(c.tctx, sql, args...)
+	// we do not retry, let user do it
```

Comment:
It's confusing that we have `retryer` in this module but not using it. Automatically retry might make life easier.

Reply:
those methods only called from dmctl, i.e. user called, which should return fast, and we don't met retryable error often, and if it really happens, user can check it.

for other methods which uses retry, it's because it's run in background, stop-then-resume makes life harder.

Reply:
lgtm